appinfo/routes.php

```diff
@@ -46,25 +46,13 @@
 	],
 	'ocs' => [
 		[
-			'name' => 'api#getRooms',
+			'name' => 'Call#getRooms',
 			'url' => '/api/{apiVersion}/room',
 			'verb' => 'GET',
 			'requirements' => ['apiVersion' => 'v1'],
 		],
 		[
-			'name' => 'api#makePublic',
-			'url' => '/api/{apiVersion}/room/public',
-			'verb' => 'POST',
-			'requirements' => ['apiVersion' => 'v1'],
-		],
-		[
-			'name' => 'api#makePrivate',
-			'url' => '/api/{apiVersion}/room/public',
-			'verb' => 'DELETE',
-			'requirements' => ['apiVersion' => 'v1'],
-		],
-		[
-			'name' => 'api#getRoom',
+			'name' => 'Call#getRoom',
```

Comment:
This method is in the room controller, not the call controller:
https://github.com/nextcloud/spreed/blob/5cbc34ac74b878ee83ad8bbe6c9b2d947cd23127/lib/Controller/RoomController.php#L122